src/core/public/public.api.md

```diff
@@ -856,6 +862,25 @@ export interface IUiSettingsClient {
     set: (key: string, value: any) => Promise<boolean>;
 }
 
+// @internal (undocumented)
+export interface LegacyApp extends AppBase {
+    // (undocumented)
+    appUrl: string;
+    // (undocumented)
+    disableSubUrlTracking?: boolean;
+    // (undocumented)
+    linkToLastSubUrl?: boolean;
+    // (undocumented)
+    subUrlBase?: string;
+}
+
+// Warning: (ae-incompatible-release-tags) The symbol "LegacyAppInfo" is marked as @public, but its signature references "LegacyApp" which is marked as @internal
```

Comment:
might want to fix this? not sure if very important though

Reply:
`LegacyApp` was flagged as internal, however `LegacyAppInfo` is based on it, and is public. What should I do, change `LegacyApp` visibility to `public`?

Reply:
I guess so. It doesn't matter a whole lot since this is going away soon.